### ISSUE TITLE
bind-key not honored in article view

### ISSUE THREAD

**Issue by pushcx** [newsboat/newsboat#200]
In `newsboat 2.11.1` I have some bound keys:

```
unbind-key o
bind-key o open-in-browser-and-mark-read
bind-key O open-in-browser
bind-key a open-all-unread-in-browser-and-mark-read
```

These keys work when the feed list and in the list of articles in a particular feed, and I see them when I hit `?` for the list of keys. When I'm viewing an individual article, the bindings for `o` and `a` don't work and are missing from the list of keys, though `O` is listed as `open-in-browser`.

These shortcuts used to work in all views and stopped working in the last month or two. I'm sorry, I don't know exactly which version they stopped working in.

Thanks for any help, and for a great newsreader.

**Comment by Minoru** [comment]
Hi! Indeed, with 2.11.1, I see the following:

* only `O` works in articlelist, `o` and `a` don't;
* only `O` is listed in help dialog for articlelist, `o` and `a` aren't.

But in 2.10 (released 8 months ago), it behaved the same!

Also:
* from what I gather with quick search over Git log, `open-in-browser-and-mark-read` was part of articlelist from the very beginning, but was *never* implemented for articleview;
* `open-all-unread-in-browser-and-mark-read` doesn't make sense for articleview, because there's only one article in it—there is no "all" to speak of.

Are you sure you didn't tweak your keybinds recently? I don't see how the ones you show could've worked in *any* Newsboat (or Newsbeuter!) version.

**Comment by pushcx** [comment]
No, I haven't touched them in months. I swear I've seen `o -> open-in-browser-and-mark-read` work, but maybe I'm imagining things.

Well, if this can't have worked I suppose it's not a bug. Perhaps it could be a feature request? I'd really like it to work. I can understand why `a` working would be a little odd.

**Comment by Minoru** [comment]
Both of them would be odd, actually. `open-in-browser-and-mark-read` is odd for articleview because if you're looking at an article, it's already marked as read—no need to do it again.

Can you explain how making these work will make your life better, or help you in any way?

**Comment by pushcx** [comment]
I don't have to think. I can hit `o` to move on from the article, secure that it's in the browser and marked. Otherwise it's `o` in articlelist but `On` in the detail view.

**Comment by Minoru** [comment]
I see. Ideally, this should be solvable with out macro key system, but it doesn't let you write a macro for a specific dialog yet, and it makes you press an additional key to invoke the macro.

I still don't wan't to add `open-in-browser-and-mark-read` to articleview, because it's main purpose—opening and marking—doesn't completely make sense there, and once added, it will be very hard to remove.

So the only remedy I have for you right now is to do something like this:
```
unbind-key o
bind-key o open-in-browser-and-mark-read articlelist
bind-key o open-in-browser articleview
```
This will let you press `o` in articlelist and `on` in articleview, which is slightly less typing.
